src/sailor/db/elasticsearch.lua

```diff
@@ -0,0 +1,130 @@
+-------------------------------------------------------------------------------------------
+-- elasticsearch.lua: DB module for connecting, querying and serching through elasticsearch
+-- This file is a part of Sailor project
+-- Copyright (c) 2016 Nikhil. R <rnikhil96@outlook.com>
+-- License: MIT
+-- http://sailorproject.org
+-------------------------------------------------------------------------------------------
+
+
+
+local elasticsearch = require "elasticsearch"
+local main_conf = require "conf.conf"
+local elastic_conf = main_conf.db[main_conf.sailor.search_database]
+
+local elastic = {}
+
+	local client = elasticsearch.client{
+	host={
```

Comment:
https://github.com/sailorproject/sailor/pull/125/commits/a77172fef75aa5b3f5106aac66bfacc720e12a6f
With that, you can add multiple hosts now